services/unifiedOnbording/aws/awsUnifiedOnbording.go

```diff
@@ -0,0 +1,100 @@
+package aws
+
+import (
+	"fmt"
+	"net/http"
+)
+
+const (
+	AwsUnifiedOnboardingResourcePath = "AwsUnifiedOnboarding"
+	UpdateVersion                    = "UpdateVersion"
+	StackConfig                      = "StackConfig"
+)
+
+type PostureManagementConfiguration struct {
+	Rulesets []int `json:"rulesets"`
+}
+
+type ServerlessConfiguration struct {
+	Enabled bool `json:"enabled"`
+}
+type IntelligenceConfigurations struct {
+	Enabled  bool  `json:"enabled"`
+	Rulesets []int `json:"rulesets"`
+}
+
+type UnifiedOnbordingRequest struct {
+	OnboardType                    string                         `json:"onboardType"`
+	FullProtection                 bool                           `json:"fullProtection"`
+	CloudVendor                    string                         `json:"cloudVendor"`
+	EnableStackModify              bool                           `json:"enableStackModify"`
+	PostureManagementConfiguration PostureManagementConfiguration `json:"postureManagementConfiguration"`
+	ServerlessConfiguration        ServerlessConfiguration        `json:"serverlessConfiguration"`
+	IntelligenceConfigurations     IntelligenceConfigurations     `json:"intelligenceConfigurations"`
+}
+
+type UnifiedOnbordingConfigurationResponse struct {
+	StackName       string     `json:"stackName" :"stack_name"`
+	TemplateUrl     string     `json:"templateUrl" :"template_url"`
+	parameters      Parameters `:"parameters"`
+	IamCapabilities []string   `json:"iamCapabilities" :"iam_capabilities"`
+}
+
+type Parameters []struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+type UnifiedOnbordingInformationResponse struct {
+	OnboardingId            string                  `json:"onboardingId"`
+	InitiatedUserName       string                  `json:"initiatedUserName"`
+	InitiatedUserId         int                     `json:"initiatedUserId"`
+	EnvironmentId           string                  `json:"environmentId"`
+	EnvironmentName         string                  `json:"environmentName"`
+	EnvironmentExternalId   string                  `json:"environmentExternalId"`
+	RootStackId             string                  `json:"rootStackId"`
+	CftVersion              string                  `json:"cftVersion"`
+	UnifiedOnbordingRequest UnifiedOnbordingRequest `json:"onboardingRequest"`
+	Statuses                Statuses                `json:"statuses"`
+}
+
+type Statuses []struct {
+	Module                    string `json:"module"`
+	Feature                   string `json:"feature"`
+	Status                    string `json:"status"`
+	StatusMessage             string `json:"statusMessage"`
+	StackStatus               string `json:"stackStatus"`
+	StackMessage              string `json:"stackMessage"`
+	RemediationRecommendation string `json:"remediationRecommendation"`
+}
+
+func (service *Service) GetConfiguration(id string) (*UnifiedOnbordingConfigurationResponse, *http.Response, error) {
```

Comment:
this is not GetConfiguration but Get(), also you nay rename id to cloudAccountId